pkg/nodeset/v1alpha1/types.go

```diff
@@ -181,6 +186,9 @@ type NodeSetStatus struct {
 	Conditions []NodeSetCondition `json:"conditions,omitempty"`
 }
 
+// +genclient=true
```

Comment:
I would fix the namespace during prototyping at least, e.g. "kube-nodeset" or something like that. Or even default.